testsuite/fuzzer/README.md

```diff
@@ -97,6 +112,11 @@ When building in the OSS-Fuzz environment, `fuzz.sh` will place the corpus archi
 - **Error Handling:** Implement robust error handling to intercept crashes or unwanted/unexpected behavior.
 - **Performance Optimization:** Optimize for performance to enable more iterations and deeper fuzzing.
 
+## fuzzer CLI
+`cargo run fuzzer <command>`
```

Comment:
will this replace `fuzz.sh`?

Reply:
We can; I haven't decided about it yet. So far, it’s helpful for accessing data structures and functionalities of other suites in Rust. Thought, most of the utilities would basically call other binaries.

Reply:
it may be confusing having two different utilities for the same purpose. can the functionality here be included in `fuzz.sh` ?

Reply:
I can call It from the script, but it seems a bit layered, let me think about it.